### ISSUE TITLE
SimCLRv1 Keras-TF implementation

### ISSUE THREAD

**Issue by mwdhont** [google-research/simclr#66]
Dear Ting,

First of all, congratz with your research, I really like the effectiveness and simplicity of the SimCLR framework. Second, I would like to share with you the keras-tf implementation I’ve made: 

https://github.com/mwdhont/SimCLRv1-keras-tensorflow

My research project schedule did not allow to do a benchmark, but within the project context I obtained some nice results on the trashnet-dataset. More specifically, SimCLR provided me with a 17% top-1 accuracy increase for a linear classifier trained on 5% of my training data. Also some nice t-SNE visualizations illustrate the added value of improving the feature encoding of the base model with SimCLR. For more details, I would like to refer to the repository.

As I tried to provide the necessary documentation, I think it should be possible to apply the framework to any Keras-model with only minor changes. So I thought it is maybe worth linking the repository?

Enjoy your day!
Michiel

**Comment by chentingpc** [comment]
thanks for sharing, it has been linked.
